llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2285,10 +2285,16 @@ class VPWidenPHIRecipe : public VPSingleDefRecipe {
   /// List of incoming blocks. Only used in the VPlan native path.
   SmallVector<VPBasicBlock *, 2> IncomingBlocks;
```

Comment:
Maybe update the comments above re: usage in vplan native?  Given this is now used in both paths, do these need delete or clarified?

Reply:
IncomingBlocks should still only be used in the vplan native path, so I've added more comments to the other methods that should only be called from it